playwright.config.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-env node */
+import { defineConfig, devices } from '@playwright/test'
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './test',
+  outputDir: './tmp/playwright-results/',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  timeout: 3000,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: [
+    [
+      'html',
+      {
+        outputFolder: './tmp/playwright-html-report/',
+      },
+    ],
+  ],
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    // baseURL: 'http://127.0.0.1:3000',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: process.env.CI ? 'on-first-retry' : 'on',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
```

Comment:
Ik heb net bij @bartjkdp nagevraagd welke browsers we willen ondersteunen:

> De laatste versie van Chrome, Firefox, Edge en Safari

Reply:
Chrome en Edge zijn praktisch hetzelfde, het lijkt me voldoende om dan met Chrome, Firefox en Webkit te testen.

Reply:
Heb ze toegevoegd